src/control/controlproxy.h

```diff
@@ -48,28 +48,19 @@ class ControlProxy : public QObject {
         // requested: Queued -> COP = Queued / SCO = Auto
         // requested: BlockingQueued -> Assert(false)
 
-        auto copSlot = &ControlProxy::slotValueChangedAuto;
-        Qt::ConnectionType copConnection;
         Qt::ConnectionType scoConnection;
         switch(requestedConnectionType) {
         case Qt::AutoConnection:
-            copConnection = Qt::AutoConnection;
+            [[fallthrough]];
```

Comment:
I think we need the [[fallthrough]]; only in case there is extra code.

Reply:
Line 56 is the extra code that the `Qt::AutoConnection` case falls through to.

Reply:
Ah sorry, no I get what you meant.